### PR TITLE
Create php-database.md

### PR DESCRIPTION
**Pull request by jonathanglassman**
## What

Created markdown file with content on how to:

- Connect a PHP app to PostgreSQL and MySQL
- Connect Drupal to MySQL
- Connect Wordpress to MySQL

This is to complete story https://www.pivotaltracker.com/story/show/153630350

## How to review

Describe the steps required to test the changes.

1. Preview the content locally ([see README](https://github.com/alphagov/paas-tech-docs#preview)) and check that it renders as expected.
1. Manually check that any removed or renamed anchor tags are not in use ([linkchecker](https://github.com/linkcheck/linkchecker) doesn't do this).
1. Check that content fulfills story criteria.

## Who can review

Anyone except Jon Glassman or Andras Ferencz-Szabo

### CODE REVIEW

File: source/documentation/guidance/php-database.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+You configure your app to enable this secure connection by inserting the following code into the `config.ini` file located within `.bp-config/php/php.ini.d/`:
+
+```
+extension=pdo.so
+extension=pdo_mysql.so OR pdo_pgsql.so
+extension=openssl.so
+```
+
+You should use this method even if you have previously defined PHP extensions in the `.bp-config/options.json` buildpack config file.
```
Comment (samcrang) [inline source/documentation/guidance/php-database.md]: Does it make sense to be explicit that this is "instead of" and not "as well as"?

Comment (jonathanglassman) [reply source/documentation/guidance/php-database.md]: yes that's a good point

**Comment by samcrang** [comment]
@jonathanglassman does the newly added file need to be referenced by `source/index.html.md.erb`?

**Comment by jonathanglassman** [comment]
Have added php-database to `source/index.html.md.erb`

**Pull request merged**